app/components/ProgressStepIndicator.tsx

```diff
@@ -37,18 +38,29 @@ const NumberedCircle: React.FunctionComponent<NumberCircleProps> = ({
 };
 
 export const ProgressStepIndicator: React.FunctionComponent<Props> = ({
-  steps
+  steps,
+  title
 }) => {
   return (
     <>
+      {title && (
+        <>
+          <Row>
+            <Col>
+              <h2>{title}</h2>
+            </Col>
+          </Row>
+          <br />
+        </>
+      )}
       <div className="progress ml-1 mr-1">
         <div
           role="progressbar"
           className="progress-bar"
           aria-valuenow={0}
           aria-valuemin={0}
           aria-valuemax={100}
-          aria-label="operator-progress-indicator"
+          aria-label="progress-step-indicator"
```

Comment:
Given that `aria-label` is read by a screen reader, I would assume that it should be a readable label, not an id